BoostTestPlugin/BoostTestPlugin.csproj

```diff
@@ -40,6 +41,7 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>BoostUnitTestAdapter</RootNamespace>
     <AssemblyName>BoostUnitTestAdapter</AssemblyName>
+    <RuntimeIdentifiers>win</RuntimeIdentifiers>
```

Comment:
Curious why we need to add this now?

Reply:
Was running into a build issue that this fixed. Not sure it's applicable now still. I'll test removing it.